sale_order_archive/readme/USAGE.rst

```diff
@@ -0,0 +1,14 @@
+To archive sale orders, you need to:
+
+#. Open the tree view of sale orders.
+#. Select a sale order (in status Locked or Cancelled) you want to archive.
```

Comment:
Just an idea, what if user archive the Sales Order which is done or in process? It should have some mechanism to prevent the user to do that.
@pedrobaeza What do you think?